packages/ui/Button/Button.js

```diff
@@ -99,13 +99,22 @@ const ButtonBase = kind({
 		 * will not be rendered.
 		 *
 		 * If this is a component rather than an HTML element string, this component will also
-		 * receive the `size` property and should be configured to handle it.
+		 * receive the `size` and `flip` properties and should be configured to handle it.
```

Comment:
This should be `iconFlip`, I think.  Or, rather, reworded to receive the `iconFlip` prop as `flip`.

Reply:
:+1: I'm on it